mapdata/WarcraftLegacies/UpgradeData/ZBHS.json

```diff
@@ -102,11 +77,11 @@
       "Level": 1,
       "Id": 828536167,
       "Type": 3,
-      "Value": "Changes Obsidian Eradicators\u0027 armor type to Heavy and replaces their Spell Resistance with Hardened, which causes them to take \u003CZBHL,DataC1\u003E reduced damage from attacks.|n|n|cffc864c8Can\u0027t be researched with Spell Conduction.|r"
+      "Value": "Increases Obsidian Eradicators maximum hit points by 200.|n|n|cffc864c8Can\u0027t be researched with Spell Conduction.|r"
```

Comment:
There's no need to get rid of the Hardened ability, I think. We just want to remove the Heavy armor conversion.